<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-1</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 221, 208)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QTextEdit" name="Code_space">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>601</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
color: black;
font: 12pt &quot;Terminal&quot;;</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Terminal'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Code example&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>619</x>
       <y>9</y>
       <width>161</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white</string>
     </property>
     <widget class="QPushButton" name="input_bt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
                border: 2px solid black;
                border-radius: 5px;
                padding: 5px;
                background-color: lightgray;
				color: black;
				font: 14pt &quot;Terminal&quot;;
            }
            QPushButton:hover {
                border-color: blue;
            }
            QPushButton:pressed {
                border-color: red;
            }</string>
      </property>
      <property name="text">
       <string>Input</string>
      </property>
     </widget>
     <widget class="QPushButton" name="output_bt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
                border: 2px solid black;
                border-radius: 5px;
                padding: 5px;
                background-color: lightgray;
				color: black;
				font: 14pt &quot;Terminal&quot;;
            }
            QPushButton:hover {
                border-color: blue;
            }
            QPushButton:pressed {
                border-color: red;
            }</string>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
     <widget class="QPushButton" name="assign_bt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
                border: 2px solid black;
                border-radius: 5px;
                padding: 5px;
                background-color: lightgray;
				color: black;
				font: 14pt &quot;Terminal&quot;;
            }
            QPushButton:hover {
                border-color: blue;
            }
            QPushButton:pressed {
                border-color: red;
            }</string>
      </property>
      <property name="text">
       <string>Assign</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ifthen_bt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
                border: 2px solid black;
                border-radius: 5px;
                padding: 5px;
                background-color: lightgray;
				color: black;
				font: 14pt &quot;Terminal&quot;;
            }
            QPushButton:hover {
                border-color: blue;
            }
            QPushButton:pressed {
                border-color: red;
            }</string>
      </property>
      <property name="text">
       <string>If-Then</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>771</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
font: 14pt &quot;Terminal&quot;;</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>731</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black</string>
      </property>
      <property name="text">
       <string>Example text</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInicio">
    <property name="title">
     <string>Inicio</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
   </widget>
   <addaction name="menuInicio"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
